sympy/series/tests/test_limits.py

```diff
@@ -1295,3 +1295,11 @@ def test_issue_23752():
     assert limit(expr1, x, 0, '-') == -sqrt(3)*I
     assert limit(expr2, x, 0, '+') == sqrt(3)*I
     assert limit(expr2, x, 0, '-') == -sqrt(3)*I
+
+
+def test_issue_24276():
+    f=Lambda(x, log(tan(pi/2*tanh(x))).diff(x))
+    assert f(x).limit(x, oo) == 2
+    assert f(x).simplify().limit(x, oo) == 2
+    assert f(x).rewrite(sin).limit(x, oo) == 2
+    assert f(x).rewrite(sin).simplify().limit(x, oo) == 2
```

Comment:
```suggestion
    fx = log(tan(pi/2*tanh(x))).diff(x)
    assert fx.limit(x, oo) == 2
    assert fx.simplify().limit(x, oo) == 2
    assert fx.rewrite(sin).limit(x, oo) == 2
    assert fx.rewrite(sin).simplify().limit(x, oo) == 2
```

Reply:
You will also need to add your name to the `.mailmap` file. Tests reported:

```
This author is not included in the .mailmap file:
Victor <chrollolucilfer[14](https://github.com/sympy/sympy/actions/runs/3598501979/jobs/6061346923#step:6:15)02@gmail.com>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file
```

Reply:
ok